src/components/scenes/ReviewTriggerTestScene.tsx

```diff
@@ -37,7 +37,7 @@ import { EdgeText } from '../themed/EdgeText'
 
 interface Props extends EdgeSceneProps<'reviewTriggerTest'> {}
 
-export const ReviewTriggerTestScene = (props: Props) => {
+export const ReviewTriggerTestScene = (props: Props): React.JSX.Element => {
```

Comment:
This should be `export const ReviewTriggerTestScene: React.FC<Props> = props => {`